### ISSUE TITLE
error "mismatched types"

### ISSUE THREAD

**Issue by weberhen** [MathGaron/pytorch_toolbox#17]
When I try to run on CPU mode I have a problem:

1) on "setup_loaded_data" it will set the data to float and target to long (line 64) , which gives me a torch error of mismatching types in the loss function. So my workaround is convert target to float and not to long.
